README.md

```diff
@@ -133,9 +132,9 @@ See [README-CONAN](README-CONAN.md) for more information about Conan.
 When you build, you may install with the following command.
 
 ```cmd
-> cmake --install .
+> cmake --install build
 ```
-This will create and copy the exiv2 build artefacts to `%ProgramFiles%/exiv2`. To be able to run the `exiv2` command line application from any terminal you should modify your path to include `%ProgramFiles%/exiv2/bin`.
+This will create and copy the exiv2 build artifacts to `%ProgramFiles%/exiv2`. To be able to run the `exiv2` command line application from any terminal you should modify your path to include `%ProgramFiles%/exiv2/bin`.
```

Comment:
Some of us have grown rather fond of the British spelling sprinkled around Mr Mills, our long term maintainer 😉